app/models/work.rb

```diff
@@ -0,0 +1,34 @@
+class Work < ApplicationRecord
+  validates :title, presence: true, uniqueness: true
+  has_many :votes
+  has_many :users, through: :votes
+
+  def vote_counter
+    return self.votes.length
+  end
+
+  def self.spotlight_work(works)
+    spotlight_work = works.first
+    works.each do |work|
+      if work.vote_counter > spotlight_work.vote_counter
+        spotlight_work = work
+      end
+    end
+    return nil if spotlight_work.vote_counter == 0
+    return spotlight_work
+  end
+
+  def self.top_ten(works, category)
```

Comment:
Similar to above, I wonder what would need to change if you simply made this method signature `def self.top_ten(category)`?